### PR TITLE
Payout dashboard loading style updated to look like PR dashboard

### PR DESCRIPTION
**Pull request by sauravmaiti22**
Updated the Payout dashboard to show loader as per new approach taken for PR dashboard.

### CODE REVIEW

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/78.

**Review by donalnofrixion — APPROVED** [review]

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/78.

File: packages/components/src/components/functional/PayoutDashboard/PayoutDashboard.tsx [hunk 1]
```diff
@@ -305,12 +320,37 @@ const PayoutDashboardMain = ({
...
+  ): 'all' | 'paid' | 'inProgress' | 'pendingApproval' | 'failed' => {
+    switch (status) {
+      case PayoutStatus.All:
+        return 'all'
+      case PayoutStatus.PENDING:
+        return 'inProgress'
+      case PayoutStatus.PROCESSED:
+        return 'paid'
+      case PayoutStatus.PENDING_APPROVAL:
+        return 'pendingApproval'
```
Comment (arifmatin) [inline packages/components/src/components/functional/PayoutDashboard/PayoutDashboard.tsx:339]: Can you change this to pending Authorise/Authorisation

**Review by arifmatin — APPROVED** [review]

File: packages/components/src/components/functional/PayoutDashboard/PayoutDashboard.tsx (hunk 1 above)
Comment (donalnofrixion) [reply packages/components/src/components/functional/PayoutDashboard/PayoutDashboard.tsx:339]: Actually don't do that. It'll break the metrics code.

**Review by donalnofrixion — APPROVED** [review]

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/78.

**Pull request merged**